elastic_panel/panel.py

```diff
@@ -56,6 +57,7 @@ class ElasticDebugPanel(Panel):
 
     name = "Elasticsearch"
     template = "elastic_panel/elastic_panel.html"
+    scripts = [static("elastic_panel/js/elastic_panel.js")]
```

Comment:
@noamkush this must be a property, otherwise the tests will fail.

Reply:
I'm not sure which tests are failing, as this passes tests (and in practice it works), but you're right that the base uses a `@property` so I'll change it.